src/ResultWriter/AnalysisWriter.cpp

```diff
@@ -200,11 +225,11 @@ void seissol::writer::AnalysisWriter::printAnalysis(double simulationTime) {
         } else {
           MPI_Recv(centerRecv, 3, MPI_DOUBLE,  errLInfRecv[i].rank, i, comm, MPI_STATUS_IGNORE);
         }
-        logInfo(mpi.rank()) << "L1  , var[" << i << "] =\t" << errL1MPI[i];
-        logInfo(mpi.rank()) << "L2  , var[" << i << "] =\t" << std::sqrt(errL2MPI[i]);
-        logInfo(mpi.rank()) << "LInf, var[" << i << "] =\t" << errLInfRecv[i].val
+        logInfo(mpi.rank()) << "L1  , var[" << i << "] =\t" << errL1MPI[i] << "	" << errL1MPI[i] / analyticalL1MPI[i];
```

Comment:
Why are you using this kind of spacing? Why no tabs? Would be also good use the same spacing everywhere?